src/nebula-hooks/__tests__/use-sorting.spec.ts

```diff
@@ -11,31 +12,31 @@ describe('use-sorting', () => {
   });
 
   describe('changeSortOrder', () => {
-    let originalOrder;
-    let column;
-    let layout;
-    let model;
-    let changeSortOrder;
-    let expectedPatches;
+    let originalOrder: number[];
+    let column: Column;
+    let layout: TableLayout;
+    let model: EngineAPI.IGenericObject;
+    let changeSortOrder: ((layout: TableLayout, column: Column) => Promise<void>) | undefined;
+    let expectedPatches: EngineAPI.INxPatch[];
 
     beforeEach(() => {
       originalOrder = [0, 1, 2, 3];
       layout = generateLayout(2, 2, 2);
```

Comment:
here the type does not match. I do not know how to fix it yet.

Reply:
<img width="722" alt="Screenshot 2022-08-02 at 00 09 42" src="https://user-images.githubusercontent.com/4471489/182254306-7c1247ef-81c7-442f-9367-9849455912aa.png">

Reply:
I will have alook